### ISSUE TITLE
Can't paste on windows platform

### ISSUE THREAD

**Issue by ar37-rs** [1Password/arboard#84]
Since version 3, arboard  can't paste on other apps with text edior including vscode.

**Comment by complexspaces** [comment]
Thanks for opening an issue, @ar37-rs. I was able to reproduce the behavior locally but AFAICT, this is a side-effect of intentional changes made in https://github.com/1Password/arboard/pull/60. Does your problem go away if you close the clipboard right after finishing the text setting?

In that PR, `arboard` switched to only opening the clipboard once per call to `Clipboard::new()`, instead of open and closing it with every operation. We did this so that so users can perform multiple operations on the clipboard "atomically" if desired. The intended behavior is that you drop a `Clipboard` structure when you're done with a set of operations [like the Win32 docs reference](https://learn.microsoft.com/en-us/windows/win32/dataxchg/using-the-clipboard#copying-information-to-the-clipboard).

I apologize for forgetting to add this in the changelog for 3.0. I've opened https://github.com/1Password/arboard/pull/85 to handle that.
